src/modules/new-request-form/fields/LookupField.tsx

```diff
@@ -0,0 +1,243 @@
+      searchParams.set("requester_id", userId.toString());
+      if (organizationId !== null)
+        searchParams.set("organization_id", organizationId);
+      setIsLoadingOptions(true);
+      try {
+        const response = await fetch(
+          `/api/v2/custom_objects/${customObjectKey}/records/autocomplete?${searchParams.toString()}`
+        );
+
+        const data = await response.json();
+        if (data !== undefined) {
+          setOptions(
+            data.custom_object_records.map(
+              ({ name, id }: { name: string; id: string }) => ({
+                name,
+                value: id,
+              })
+            )
+          );
+        } else {
+          setOptions([]);
+        }
+      } catch (error) {
+        return error;
+      } finally {
+        setIsLoadingOptions(false);
+      }
+      return;
+    },
+    [customObjectKey, fieldId, organizationId, userId]
+  );
+
+  const debouncedFetchOptions = useMemo(
+    () => debounce(fetchOptions, 300),
+    [fetchOptions]
+  );
+
+  useEffect(() => {
+    return () => debouncedFetchOptions.cancel();
+  }, [debouncedFetchOptions]);
+
+  const handleChange = useCallback<NonNullable<IComboboxProps["onChange"]>>(
+    ({ inputValue, selectionValue }) => {
+      if (selectionValue !== undefined) {
+        if (selectionValue == "") {
+          setSelectedOption(EMPTY_OPTION);
+          setInputValue(EMPTY_OPTION.name);
+          setOptions([]);
+          onChange(EMPTY_OPTION.value);
+        } else {
+          const selectedOption = options.find(
+            (option) => option.value === selectionValue
+          );
+          if (selectedOption) {
+            setInputValue(selectedOption.name);
+            setSelectedOption(selectedOption);
+            setOptions([selectedOption]);
+            onChange(selectedOption.value);
+          }
+        }
+        return;
+      }
+
+      if (inputValue !== undefined) {
+        setInputValue(inputValue);
+        debouncedFetchOptions(inputValue);
+      }
+    },
+    [debouncedFetchOptions, onChange, options]
+  );
+
+  useEffect(() => {
+    if (value) {
+      fetchSelectedOption(value as string);
+    }
+  }, []); //we don't set dependency array as we want this hook to be called only once
+
+  const onFocus = () => {
+    setInputValue("");
+    selectedOption && setOptions([selectedOption]);
+  };
+
+  return (
+    <GardenField>
+      <Label>
+        {label}
+        {required && <Span aria-hidden="true">*</Span>}
+      </Label>
+      {description && (
+        <Hint dangerouslySetInnerHTML={{ __html: description }} />
+      )}
+      <Combobox
+        inputProps={{ required }}
+        startIcon={<SearchIcon />}
+        validation={error ? "error" : undefined}
+        inputValue={inputValue}
+        selectionValue={selectedOption?.value}
+        isAutocomplete
+        placeholder={t(
+          "new-request-form.lookup-field.placeholder",
+          `Search ${label.toLowerCase()}`
```

Comment:
This is not the correct way to interpolate a string in the translation, we need pass the value as options as we are doing [here](https://github.com/zendesk/copenhagen_theme/blob/d36cd9f8dde4eea5814b61952a3c4de2426fcdaf/src/modules/new-request-form/fields/cc-field/CcField.tsx#L104). Reference: https://www.i18next.com/translation-function/interpolation

Reply:
ok, I changed that